### PR TITLE
Han60 QMK / VIA Support

### PR DESCRIPTION
**Pull request by farhandsome**
## Description

Adding QMK / VIA support for the upcoming keyboard Han60.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/han60/rules.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/han60/rules.mk:21]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/han60/han60.h [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	K41, K42, K43,                K44,                K47, K48, K4A, K4C  \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+	{ K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D }, \
+	{ K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, XXX, K3B, XXX, K3D }, \
+	{ XXX, K41, K42, K43, K44, XXX, XXX, K47, K48, XXX, K4A, XXX, K4C, K4D }  \
+}
+
+#define LAYOUT_tsangan_ISO( \
```
Comment (zvecr) [inline keyboards/han60/han60.h:75]:
```suggestion
#define LAYOUT_tsangan_iso( \
```

File: keyboards/han60/han60.h [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+	K41, K42, K43,                K44,                     K47, K49, K4B  \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+	{ K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D }, \
+	{ K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, XXX, XXX }, \
+	{ K40, K41, K42, K43, K44, XXX, XXX, K47, XXX, K49, XXX, K4B, XXX, XXX }  \
+}
+
+#define LAYOUT_tsangan_ISO_arrowkeys( \
```
Comment (zvecr) [inline keyboards/han60/han60.h:89]:
```suggestion
#define LAYOUT_tsangan_iso_arrowkeys( \
```

File: keyboards/han60/info.json [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+                { "x": 1.5, "y": 4 },
+                { "w": 1.5, "x": 2.5, "y": 4 },
+                { "w": 7, "x": 4, "y": 4 },
+                { "x": 11, "y": 4 },
+                { "x": 12, "y": 4 },
+                { "x": 13, "y": 4 },
+                { "x": 14, "y": 4 }
+            ]
+        },
+        "LAYOUT_tsangan_ISO": {
```
Comment (zvecr) [inline keyboards/han60/info.json:207]:
```suggestion
        "LAYOUT_tsangan_iso": {
```

File: keyboards/han60/info.json [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+                { "w": 1.5, "x": 0, "y": 4 },
+                { "x": 1.5, "y": 4 },
+                { "w": 1.5, "x": 2.5, "y": 4 },
+                { "w": 7, "x": 4, "y": 4 },
+                { "w": 1.5, "x": 11, "y": 4 },
+                { "x": 12.5, "y": 4 },
+                { "w": 1.5, "x": 13.5, "y": 4 }
+            ]
+        },
+        "LAYOUT_tsangan_ISO_arrowkeys": {
```
Comment (zvecr) [inline keyboards/han60/info.json:272]:
```suggestion
        "LAYOUT_tsangan_iso_arrowkeys": {
```

File: keyboards/han60/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# han60
+
+![han60](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/han60/readme.md:5]: Image and description needs updating

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/han60/rules.mk [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/han60/rules.mk:17]:
```suggestion
```

**Pull request merged**